authzed/api/v1/experimental_service.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+package authzed.api.v1;
+
+option go_package = "github.com/authzed/authzed-go/proto/authzed/api/v1";
+option java_package = "com.authzed.api.v1";
+
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+import "authzed/api/v1/core.proto";
+
+// ExperimentalService exposes a number of APIs that are currently being
+// prototyped and tested for future inclusion in the stable API.
+service ExperimentalService {
+
+  // BulkLoadRelationships is a faster path to writing a large number of
+  // relationships at once. It is both batched and streaming. For maximum
+  // performance, the caller should attempt to write relationships in as close
+  // to relationship sort order as possible: (resource.object_type,
+  // resource.object_id, relation, subject.object.object_type,
+  // subject.object.object_id, subject.optional_relation)
+  // 
+  // EXPERIMENTAL
+  rpc BulkLoadRelationships(stream BulkLoadRelationshipsRequest)
+    returns (BulkLoadRelationshipsResponse) {
+      option (google.api.http) = {
+        post: "/v1/experimental/relationships/bulkload"
+        body: "*"
+      };
+    }
+}
+
+// BulkLoadRelationshipsRequest represents one batch of the streaming
+// BulkLoadRelationships API. The maximum size is unlimited, but optimal size
+// should be determined by the calling client.
+message BulkLoadRelationshipsRequest {
```

Comment:
> The maximum size is unlimited

I think we should still set a limit

Reply:
I think the issue is that the number is going to vary wildly based on:
1) datastore choice
2) datastore scaling parameters
3) compressibility of the data on the wire
4) how many concurrent writers we're using
5) how far we are from the receiving servers

This is the reason why I left it as "an exercise for the reader". I don't think there's any way to come up with those numbers except experimentally. Setting any kind of limit will just be limiting our throughput in many cases for no reason.

Reply:
I think we shouldn't say "unlimited" here then; "depends on the datastore" might be better